src/hermes_claude_agent_sdk/__init__.py

```
@@ -1,21 +1,69 @@
 
-    This intentional no-op keeps the initial repository installable while the
-    AgentRuntime v1 implementation and compatibility handshake are reviewed.
-    No SDK client, credential lookup, subprocess, or model query occurs here.
+    The host validates the descriptor before retaining ``create_runtime``.  No
+    SDK client, credential lookup, subprocess, or model query occurs here.
     """
 
-    del context
+    register_profile = getattr(context, "register_provider_profile", None)
+    if not callable(register_profile):
+        raise TypeError("Hermes plugin context lacks register_provider_profile()")
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)